lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Inferno
+  module Sequence
+    class BulkDataPatientExportSequence < SequenceBase
+      group 'Bulk Data Patient Export'
+
+      title 'Patient Tests'
+
+      description 'Verify that Patient resources on the Bulk Data server follow the Bulk Data Access Implementation Guide'
+
+      test_id_prefix 'Patient'
+
+      requires :token
+      conformance_supports :Patient
+
+      # export
+      def export_kick_off(klass)
+        headers = { accept: 'application/fhir+json', prefer: 'respond-async' }
+
+        url = ''
+        url += "/#{klass}" if klass.present?
+        url += '/$export'
+
+        @client.get(url, @client.fhir_headers(headers))
+      end
+
+      details %(
+
+        The #{title} Sequence tests `#{title}` operations.  The operation steps will be checked for consistency against the
+        [Bulk Data Access Implementation Guide](https://build.fhir.org/ig/HL7/bulk-data/)
+
+      )
+
+      @resources_found = false
+
+      test 'Server rejects $export request without authorization' do
+        metadata do
+          id '01'
+          link 'https://build.fhir.org/ig/HL7/bulk-data/export/index.html#bulk-data-kick-off-request'
+          desc %(
+          )
+          versions :stu3
+        end
+
+        @client.set_no_auth
+        skip 'Could not verify this functionality when bearer token is not set' if @instance.token.blank?
+
+        reply = export_kick_off('Patient')
+        @client.set_bearer_token(@instance.token)
+        assert_response_unauthorized reply
+      end
+
+      test 'Server shall return "202 Accepted" and "Content-location" for $export operation' do
+        metadata do
+          id '02'
+          link 'https://build.fhir.org/ig/HL7/bulk-data/export/index.html#bulk-data-kick-off-request'
+          desc %(
+          )
+          versions :stu3
+        end
+
+        reply = export_kick_off('Patient')
+
+        # Shall return 202
```

Comment:
If this comment is necessary to explain what the assertion is asserting, then `assert_response_code` should be used instead so that you can be explicit about what status you're accepting and won't need a comment to explain the assertion.